packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4944,6 +4972,48 @@ export default class Meeting extends StatelessWebexPlugin {
     }
   }
 
+  /**
+   * This method will toggle the highlights for the current meeting if the meeting has enabled/supports Webex Assistant
+   * @param {Boolean} activate flag to enable/disable highlights
+   * @param {Object} options object with spokenlanguage setting
+   * @public
+   * @returns {Promise<void>} a promise to open the WebSocket connection
+   */
+  public async toggleHighlighting(activate: boolean, options?: {spokenLanguage?: string}) {
```

Comment:
Rather than a separate `stop` or `start` method, we can use `toggle` which enabled users to toggle the highlighting setting. This way we can also compact all the logic inside of this method.